modules/integration/tests-integration/tests-patches/src/test/resources/testng.xml

```diff
@@ -220,5 +220,11 @@
         <!--</classes>-->
     <!--</test>-->
 
+    <test name="ESBJAVA_4565" preserve-order="true" verbose="2">
+        <classes>
+            <class name="org.wso2.carbon.esb.registry.task.ESBJAVA4565TestCase"/>
+        </classes>
+    </test>
+
```

Comment:
New line prior to last xml tag is already followed by the original file as it is improve the readability.